paws_collector.js

```diff
@@ -673,7 +673,8 @@ class PawsCollector extends AlAwsCollector {
     
     reportCollectionDelay(lastCollectedTs, callback) {
         const nowMoment = moment();
-        const lastCollectedMoment = moment(lastCollectedTs);
+        // converting the time from seconds to milliseconds . 
+        const lastCollectedMoment = !isNaN(lastCollectedTs) && lastCollectedTs.toString().length < 13 ? moment(parseInt(lastCollectedTs * 1000))  :  moment(lastCollectedTs);
```

Comment:
I'd rather suggest we use seconds precision timestamps to calculate collection delay.
I don't think we need to measure collection delay at a milliseconds precision.

Reply:
we are measuring collection delay base on seconds only as [here](https://github.com/alertlogic/paws-collector/pull/323/files#diff-82fd638370f7c86cddbb67ebf81bc2952ba84a1e7dc12ce1ed0fe1da30045492L678). 

Only i am converting seconds to milliseconds to get correct date . moment(lastCollectedTs) , for cisco duo  lastCollectdTs is unix() date so we need conversion. Other wise return wrong date 
ex. 
moment(1675435692) = Tue Jan 20 1970 09:23:55 GMT+0000

Reply:
you're right.
In this case I suggest we use `moment.unix()` to convert timestamp from Unix seconds, like
```
lastCollectedTs.toString().length == 10 ? moment.unix(lastCollectedTs) ?  moment(lastCollectedTs);
```